onnxruntime/core/mlas/lib/qgemm.cpp

```diff
@@ -406,7 +406,9 @@ Return Value:
     const size_t AlignedBytesRequired = (BytesRequired + BufferAlignment - 1) &
         ~(BufferAlignment - 1);
     //If this gemm B argument is used in a dynamically quantization gemm operation we can optimize for
-    //this use case. Concat both packed representations for later decision.
+    //this use case. Concat both packed representations for later decision. This allows for cases later
+    //where can_use_dynamic_mlas is false we still have the prepack at the cost of some memory otherwise
```

Comment:
this comment about a private data member in another source file outside of MLAS seems out of place. it could be documented more at the call site if needed.